src/main/utils/upload.ts

```diff
@@ -0,0 +1,118 @@
+import Store, { Schema } from 'electron-store';
+import path from 'path';
+import * as fs from 'promise-fs';
+import { FILE_STREAM_CHUNK_SIZE } from '../../config';
+import { ElectronFile, StoreType } from '../types';
+
+export const uploadStoreSchema: Schema<StoreType> = {
+    filePaths: {
+        type: 'array',
+        items: {
+            type: 'string',
+        },
+    },
+    collectionName: {
+        type: 'string',
+    },
+};
+
+const store = new Store({
```

Comment:
let's create a store variable and name it something like `uploadStatusStore` and export it everywhere so that we don't have setup it everywhere